BinaryTree.java

```diff
@@ -0,0 +1,165 @@
+package com.katermar;

```

Comment:
It is OK to leave assignment 2 on current branch and merge it to master once review is finished. Moving it to a new branch would be a bit problematic and would probably lose review comments.
But for assignment 3 (and 4 and so on) please create new branches. Thanks!